packages/overlay/src/integrations/sentry/utils/sdkValidation.ts

```diff
@@ -0,0 +1,11 @@
+import { SDK_VERSION_SUPPORT_REGEX } from '../constants';
+import { WindowWithSentry } from '../types';
+
+export function checkBrowserSDKVersion() {
+  const spotlightV1SupportedSentryRegex = new RegExp(SDK_VERSION_SUPPORT_REGEX);
```

Comment:
l: I'd rather just inline the regex here instead of initializing a new `RegExp` with the constant. Makes things a bit simpler and as far as I can tell, we're not reusing the regex.

Reply:
yup, sure